recipe/meta.yaml

```diff
@@ -24,6 +24,16 @@
 {% set PY_VER_MAJOR = PY_VER.split('.')[0] %}
 {% set PY_VER_MINOR = PY_VER.split('.')[1] %}
 
+{% set build = 10 %}
+
+{% if qt_version == "5" %}
+{% set build = build + 200 %}
+{% endif %}
+
+{% if qt_version == "6" %}
+{% set build = build + 400 %}
+{% endif %}
```

Comment:
Any particular reason for 200/400? I'm thinking we could do 500/600 for qt 5/6, that way the qt version would be relatively easy to read off from the build number, and 100 builds should be enough for the lifetime of a given version I believe.

Reply:
changed it. i didn't want really high numbers. but ok.